src/administration/from_scratch.md

```diff
@@ -209,6 +334,55 @@ sed -i -e 's/{{lemmy_ui_port}}/1234/g' /etc/nginx/sites-enabled/lemmy.conf
 nginx -s reload
 ```
 
+If you don't need HTTPS on nginx, the below file can be used as an updated template. Otherwise, feel free to merge changes from it to your own version.
```

Comment:
HTTPS is mandatory for federation so I dont think this is useful.

Reply:
@Nutomic 

Noted on HTTPS and federation.

Could you post a link to the Nginx vhost template for Lemmy? I might be wrong, but I think the one in the repo does not work out of the box, because the rewrite rules for API vs UI content are not present there. At least I had to copy-paste them from somewhere else.